internal/attestation/idkeydigest/idkeydigest.go

```diff
@@ -4,17 +4,107 @@ Copyright (c) Edgeless Systems GmbH
 SPDX-License-Identifier: AGPL-3.0-only
 */
 
+// Package idkeydigest contains policies and type definitions
+// for checking the ID Key Digest value in SEV-SNP attestation.
 package idkeydigest
 
 import (
 	"encoding/hex"
 	"encoding/json"
 	"errors"
 	"fmt"
+	"strings"
 
 	"github.com/edgelesssys/constellation/v2/internal/cloud/cloudprovider"
 )
 
+//go:generate stringer -type=EnforceIDKeyDigest
+
+// EnforceIDKeyDigest defines the behavior of the validator when the ID key digest is not found in the expected list.
+type EnforceIDKeyDigest uint32
+
+// TODO: Decide on final value naming.
```

Comment:
we should also think about renaming the key, but not sure what this means for backwards compat

Reply:
Would be a breaking change from my pov. Maybe we can rename it when implementing #1436?
For that one we will need migration docs/code anyways.